mailpoet/lib/EmailEditor/Engine/Renderer/Preprocessors/BlocksWidthPreprocessor.php

```diff
@@ -7,11 +7,16 @@
  * The final width in pixels is stored in the email_attrs array because we would like to avoid changing the original attributes.
  */
 class BlocksWidthPreprocessor implements Preprocessor {
+
+  const BLOCKS_WITHOUT_WIDTH = [
+    'core/paragraph',
+  ];
```

Comment:
I'm afraid in the future when we add third-party blocks or universal renderers, we won't be able to have such constants, or we will have to require a third party to give us the information if the block has or hasn't width.
Could we make it work without such a list?

Reply:
That is a good point. I guess we can manage this in the paragraph block renderer.